smoke-tests/spec/external_dns_spec.rb

```diff
@@ -1,39 +1,44 @@
 require "spec_helper"
 
 describe "external DNS" do
-  # let!(zone) { nil }
+  namespace = "child-#{readable_timestamp}"
```

Comment:
Can we have "integrationtest" in the namespace name? That matches the other tests, and will make it easy for us to identify and cleanup any leftovers (e.g. from crashed test runs)